nunjucks/src/precompiled-loader.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const Loader = require('./loader');
+const path = require('path').posix;
```

Comment:
this loader is used in `web-loaders`, and this dependency will likely break browser bundle (not checked this)